README.md

```diff
@@ -92,7 +91,11 @@ All notable changes to this project will be documented in the [CHANGELOG.md](htt
 
 ## Authors
 
-See the [AUTHORS](https://github.com/lomsa-dev/http-mock-adapter/blob/master/AUTHORS "Project's AUTHORS file") file for information regarding the authors of the project.
```

Comment:
Why remove this entry? If you intended to replace _Authors_ with _Acknowledgments_, the header/title should be changed too.

Reply:
My bad, i'll fix.

Reply:
Done